tools/validate_configs/test_configs/htcondor-pool.yaml

```diff
@@ -0,0 +1,117 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+blueprint_name: htcondor-pool
+
+vars:
+  project_id:  ## Set GCP Project ID Here ##
+  deployment_name: htcondor-001
+  region: us-central1
+  zone: us-central1-c
+  image_family: htcondor-9x
+  network_name: htcondor-pool
+  subnetwork_name: htcondor-pool-usc1
+
+deployment_groups:
+- group: htcondor-env
+  modules:
+  - source: modules/network/vpc
+    kind: terraform
+    id: network1
+    outputs:
+    - network_name
+  - source: community/modules/scripts/htcondor-install
+    kind: terraform
+    id: htcondor_install
+  - source: community/modules/project/service-enablement
+    kind: terraform
+    id: htcondor_services
+    use:
+    - htcondor_install
+  - source: modules/scripts/startup-script
+    kind: terraform
+    id: htcondor_install_scripts
+    settings:
+      runners:
+      - type: shell
+        source: modules/startup-script/examples/install_ansible.sh
+        destination: install_ansible.sh
+      - $(htcondor_install.install_htcondor_runner)
+    outputs:
+    - startup_script
+- group: packer
+  modules:
+  - source: modules/packer/custom-image
```

Comment:
My understanding: you are building htcondor install into a custom image. Is this required or just an optimization. It might make the example considerably clearer to just do the htcondor install on boot even if it is a little slower, and then we can always point people to the packer example if they want to optimize. 

Especially since the example does not actually work without the special passing of startup script between group 1 and 2, which you probably don't want to document again.

Reply:
This is a test configuration, not an example. Its intent is to be the basis for an integration test that builds an image and provisions an HTCondor pool from it.

I agree that a user-facing example faces the challenge you describe. When full HTCondor functionality is ready, I will produce a suitably simple example.

Reply:
🤷‍♂️ It seems like the eventual integration test will also be simpler if it were laid out as I describe, but your call. Feel free to close.

Reply:
I think building an image tests a production-like deployment and is less prone to transient failures (yum install must succeed only once vs O(N) times).

Reply:
I also meant to add that your comment made me take a look back at the `htcondor-install` module. I need to double check that it is idempotent with respect to firewall rules and multiple boot cycles. I consider that a separate task.

Reply:
Task created